.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -38,9 +38,7 @@ jobs:
           - kata-ctl
           - kernel
           - kernel-confidential
-          - kernel-sev
```

Comment:
`tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh` still has some references to them. Should we remove there too?

Reply:
Very good point, @beraldoleal, and I should have written this as part of a note in the PR.
If we remove those right now, we'll break the build as:
* removing from the yaml file will only take effect after the PR is merged
* removong from the kata-deploy-binaries.sh will take effect during the tests

With this in mind, I'd like to remove those bits in a follow-up PR.

Does it sound reasonable?

Reply:
Sounds acceptable! :) Thanks.